### ISSUE TITLE
Wrong default work experience selected

### ISSUE THREAD

**Issue by harshgoel05** [harshgoel05/another-portfolio#90]
**Describe the bug**
The default work experience selected is not from the list of featured work experiences

**To Reproduce**
1. Go to https://harshgoel.me/work
3. Scroll down to the end of page
4. See an error

**Expected behavior**
The first 0th index should come up

**Screenshots**
![image](https://user-images.githubusercontent.com/48649088/135571225-23810dd6-6fe7-45f7-bae4-1afdebd05aac.png)

**Comment by dulcetsoul** [comment]
I want to give this issue a try. Shall I?
As far I understood the issue, is with the number appearing behind the box. Please confirm me whether I understood it correct ? Thank you :)

**Comment by harshgoel05** [comment]
No, that's fine. If you see "Zoob Labs" isn't on the list but it's selected by default.

It's due to the API gives an array of objects which has a key called "featured", if featured: true it goes in the list. If not, it doesn't.

So, you just need to apply the same when showing it in the box/ set active state default in useState.

@dulcetsoul

**Comment by harshgoel05** [comment]
It will be in the [line 54 in work/details.tsx](https://github.com/harshgoel05/another-portfolio/blob/66ebb17f42a3206e59966d3021c878a50282efc1/components/work/details.tsx#L54)

Consider these two lines:
```
activeCompany={companyDetails[activeIndex]}
```

```
 const [activeIndex, setActiveIndex] = useState(0);  
```

The default goes 0 in [line 10 ](https://github.com/harshgoel05/another-portfolio/blob/66ebb17f42a3206e59966d3021c878a50282efc1/components/work/details.tsx#L10) so changing this will work

**Comment by harshgoel05** [comment]
@dulcetsoul Still working?

**Comment by GaushJ** [comment]
Hey @harshgoel05 If this issue still persists . Can you assign this to me?

**Comment by harshgoel05** [comment]
Go ahead @GaushJ!
Also make sure you use gitmoji for commit messages.

**Comment by GaushJ** [comment]
Hey @harshgoel05 . I didn't clearly understand the problem you stated . Do you want your list to be started from 1 instead of 0? 
Here's what I did.
![sent](https://user-images.githubusercontent.com/53349226/135766373-a8a15de6-f663-48a3-af55-548ecefcdfca.PNG)

**Comment by harshgoel05** [comment]
No, that was all right. You don't need to do that.

**Comment by GaushJ** [comment]
Could you elaborate your problem ? what I learnt is  that the API retrieves an array of objects and the condition checks if the company is featured . But the issue is with the order in which it retrieves . Am I right ?

**Comment by GaushJ** [comment]
Hey @harshgoel05 could you help me with the above issue?

**Comment by harshgoel05** [comment]
The issue is simple,  ` the condition checks if the company is featured .` is not applicable for the box one, its hardcoded for 0th index  as mentioned https://github.com/harshgoel05/another-portfolio/issues/90#issuecomment-932342298

**Comment by rahul-ghimire-au6** [comment]
@harshgoel05 is this issue solved ?? if yes can you please close it?
